packages/vx-axis/src/axis/AxisBottom.tsx

```diff
@@ -4,16 +4,19 @@ import Axis from './Axis';
 import Orientation from '../constants/orientation';
 import { SharedAxisProps, AxisScale } from '../types';
 
-export default function AxisBottom<Scale extends AxisScale>({
-  axisClassName,
-  labelOffset = 8,
-  tickLabelProps = (/** tickValue, index */) => ({
+export const bottomTickLabelProps = (/** tickValue, index */) =>
```

Comment:
exporting these so that they could be used outside the component. tick label props are the primary differences between axis configurations